sklearn/metrics/_plot/roc_curve.py

```diff
@@ -374,11 +374,10 @@ def plot_roc_curve(
 
     Read more in the :ref:`User Guide <visualizations>`.
 
-    .. deprecated:: 1.0
-      `plot_roc_curve` is deprecated in 1.0 and will be removed in
-       1.2. Use one of the following class methods:
-       :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
-       :func:`~sklearn.metrics.RocCurveDisplay.from_estimator`.
+    `plot_roc_curve` is deprecated in 1.0 and will be removed in
+    1.2. Use one of the following class methods:
+    :func:`~sklearn.metrics.RocCurveDisplay.from_predictions` or
+    :func:`~sklearn.metrics.RocCurveDisplay.from_estimator`.
```

Comment:
@ogrisel do you think that we can remove this part since it will be displayed twice in the documentation. In case that we check the documentation in `ipython`, the warning will be shown at import.

Reply:
+1 for removing.